### PR TITLE
Mission/Values Page Redesign / Refactor

### PR DESCRIPTION
**Pull request by travisdmathis**
##### Description

- Implements New Design for Missions/Value Page
- Refactors HTML to 2 space Indent
- Refactors for char line limit.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)

- html
- css

##### Refers/Fixes

Fixes: #1220

### CODE REVIEW

**Comment by thelostone-mc** [comment]
@travisdmathis could we make the heading bolder (like in the design )

screen shot 2018-05-25 at 5 23 37 pm


Could we remove the white space , it looks off 

screen shot 2018-05-25 at 5 24 12 pm

**Comment by travisdmathis** [comment]
@thelostone-mc yep will take care of it shortly, as a side note, you might want to check throughout the app where you're using Futura font.. There was a bug in this page where they were using Futura instead of font-family: futura-pt.. Which was causing it to fall back to sans-serif.

**Comment by travisdmathis** [comment]
@thelostone-mc pushed requested changes.

**Comment by thelostone-mc** [comment]
Ah thanks for pointing that out @travisdmathis !! Will fix it up

**Comment by travisdmathis** [comment]
@thelostone-mc need a re-review, greened up CI.

**Comment by mbeacom** [comment]
@travisdmathis 🙌 Thanks! I was just about to push a fix for that.  Reviewing now!

File: app/retail/templates/mission.html [hunk 1]
```diff
@@ -26,212 +26,356 @@
...
+  <div class="row animatedParent we-want">
+    <div class="col-md-12 col-lg-8 col-xl-6 animated fadeInLeftShort">
+      <div class="row mb-5">
+        <div class="col-md-10 col-lg-8 d-flex align-items-center">
+          <img
+            class="icon"
+            src="{% static "v2/images/mission/want/rocket_man.svg" %}"
+          />
+          <span class="icon-text">
+            We want to build a world in which anyone has the financial
```
Comment (mbeacom) [inline app/retail/templates/mission.html]: Can we wrap these remaining strings in `trans` and `blocktrans` where appropriate?

File: app/retail/templates/mission.html [hunk 2]
```diff
@@ -144,7 +150,9 @@ <h4 class="text-left text-uppercase">
 
           <div class="divider"></div>
 
+          {% blocktrans %}
           <p>Intelectual honesty</p>
```
Comment (mbeacom) [inline app/retail/templates/mission.html]: Can we fix the spelling here to `Intellectual` - Sorry, missed this the first time!

File: app/retail/templates/mission.html [hunk 3]
```diff
@@ -154,8 +162,9 @@ <h4 class="text-left text-uppercase">
           />
 
           <div class="divider"></div>
-
+          {% blocktrans %}
           <p>Intelectual honesty</p>
```
Comment (mbeacom) [inline app/retail/templates/mission.html]: Also, is this a duplicate of the above? If so, we might as well drop it.  If not, can we update the spelling here as well?

File: app/retail/templates/mission.html [hunk 4]
```diff
@@ -275,9 +297,11 @@ <h4 class="text-left text-uppercase">
             class="icon"
             src="{% static "v2/images/mission/interact/head.svg" %}"
           />
+          {% blocktrans %}
           <span class="icon-text">
             Be thoughtfull, clear, and direct.
```
Comment (mbeacom) [inline app/retail/templates/mission.html]: Can we adjust this to `Be thoughtful` ?

File: app/retail/templates/mission.html [hunk 5]
```diff
@@ -299,9 +325,11 @@ <h4 class="text-left text-uppercase">
             class="icon"
             src="{% static "v2/images/mission/interact/goal.svg" %}"
           />
+          {% blocktrans %}
           <span class="icon-text">
             Callenge the status quo / Be willing to be changed.
```
Comment (mbeacom) [inline app/retail/templates/mission.html]: `Challenge`

File: app/retail/templates/mission.html [hunk 6]
```diff
@@ -323,9 +353,11 @@ <h4 class="text-left text-uppercase">
             class="icon"
             src="{% static "v2/images/mission/interact/microscope.svg" %}"
           />
+          {% blocktrans %}
           <span class="icon-text">
             Identify and validate your asumptions.
```
Comment (mbeacom) [inline app/retail/templates/mission.html]: `assumptions`

File: app/retail/templates/mission.html [hunk 7]
```diff
@@ -359,9 +395,11 @@ <h4 class="text-left text-uppercase">
             class="icon"
             src="{% static "v2/images/mission/interact/swiss_army.svg" %}"
           />
+          {% blocktrans %}
           <span class="icon-text">
             Pragmatic over dogmatism.
```
Comment (mbeacom) [inline app/retail/templates/mission.html]: Should we update this to `Pragmatism over dogmatism`?

File: app/retail/templates/mission.html (hunk 1 above)
Comment (travisdmathis) [reply app/retail/templates/mission.html]: @mbeacom done.. wrapped all p/span strings

File: app/retail/templates/mission.html (hunk 2 above)
Comment (travisdmathis) [reply app/retail/templates/mission.html]: fixed

File: app/retail/templates/mission.html (hunk 7 above)
Comment (travisdmathis) [reply app/retail/templates/mission.html]: I think so.

File: app/retail/templates/mission.html (hunk 6 above)
Comment (travisdmathis) [reply app/retail/templates/mission.html]: Fixed

File: app/retail/templates/mission.html (hunk 5 above)
Comment (travisdmathis) [reply app/retail/templates/mission.html]: fixed

File: app/retail/templates/mission.html (hunk 4 above)
Comment (travisdmathis) [reply app/retail/templates/mission.html]: Fixed

File: app/retail/templates/mission.html (hunk 3 above)
Comment (travisdmathis) [reply app/retail/templates/mission.html]: It's a type-o, should have been Collaborative. Fixed.

**Comment by mbeacom** [comment]
@travisdmathis Apologies for all of the one-off comments.  I figured they were the **last** in each case.  I should have just put them all in a review comment.

Thanks for fixing this up!

**Comment by owocki** [comment]
guys i <3 this thanks for shipping it @travisdmathis !!

**Pull request merged**